src/index.js

```diff
@@ -3,10 +3,46 @@ import './fontawesome/css/all.css';
 import { createTodos } from './htmlDisplay.js';
 import { TaskStorage } from './localStorage.js';
 
-const todo = document.querySelector('.todo');
+/* eslint-disable */
+document.querySelector('.form-btn').addEventListener('click', (e) => {
+  const textField = document.querySelector('input[type="text"]');
+  const tasks = {
+  description: textField.value,
+  completed: false,
+  index: 0,
+  }
```

Comment:
- [ ] This expression assigns the index value of a newly created Todo item to be zero. I think it should be one plus the last element index. Kindly fix this. :pray: 

![Screenshot from 2021-12-03 12-59-52](https://user-images.githubusercontent.com/64894927/144583970-7a253be9-67f5-4f6d-a69c-88ee0aff941f.png)